Components/Comments.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { getArticleComments } from "../src/utils/api"
+
+
+const Comments = ({articleId}) => {
+
+    const [comments, setComments] = useState([])
+
+    useEffect(() => {
+        getArticleComments(articleId).then((commentsFromApi) => {
+            setComments(commentsFromApi)
+        })
+    }, [articleId])
+
+
+return (
+    <section>
+        <h2>Comments</h2>
+        <ol>
```

Comment:
Though it's great you've rendered a list of comments, I would maybe reconsider a **numbered** list of comments 🔢 
Look at sites like Youtube or Reddit for inspiration about what info is useful to display, things like the username, when it was posted for example

Reply:
okay, i will effect this now. thank you